src/demo-app/stepper/stepper-demo.html

```diff
@@ -40,7 +42,7 @@ <h3>Linear Vertical Stepper Demo using a single form</h3>
         <button mat-button>Done</button>
       </div>
     </mat-step>
-  </mat-vertical-stepper>
+  </mat-stepper>
 </form>
 
 <h3>Linear Horizontal Stepper Demo using a different form for each step</h3>
```

Comment:
The rest of this file should be updated with the new stepper.

Reply:
Not necessarily. The separate horizontal/vertical steppers are still valid use cases.

Reply:
Got it, that makes sense.